helm/ffc-demo-claim-service/values.yaml

```diff
@@ -39,12 +39,12 @@ container:
   readOnlyRootFilesystem: true
   allowPrivilegeEscalation: false
   port: 3003
-  command: ["/bin/sh","-c"]
-  args: ["npm --no-update-notifier run migrate && node index"]
   calculationQueueAddress: calculation
   claimQueueAddress: claim
   messageQueueHost: myservicebus.servicebus.windows.net
   scheduleQueueAddress: schedule
+  command: ["/bin/sh","-c"]
+  args: ["node index"]
```

Comment:
This isn't any different to the command that the Dockerfile specifies so doesn't need to be here.  It would only need to change if we wanted to deploy the development image to a cluster so the watch didn't start but we've agreed not to support local development in Kubernetes.

Reply:
Well, that's what you'd think. When I removed these lines completely the helm chart was still deploying the command with the `--no-update-notifier run migrate`, rather than the container default command, even though there was no mention of migrate in the source code. This happened on build 8 through 11.
I'll try taking the lines back out to see if whatever was cached in Helm/kubernetes has disappeared.